app/assets/stylesheets/layout_components/forms.scss

```diff
@@ -63,6 +63,84 @@ form {
   }
 }
 
+/* ------------- Switch (Toggle Button) Styles ------------- */
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 64px;
+  height: 30px;
+  float: right;
+
+  input:checked + .switch-slider {
+    background-color: $brand-green;
+
+    .switch-slider__icon {
+      &--left {
+        @extend .switch-slider__icon;
+        opacity: 1;
+        transition-delay: 0.15s;
+      }
+
+      &--right {
+        @extend .switch-slider__icon;
+        opacity: 0;
+        transition-delay: 0s;
+      }
+    }
+
+    &:before {
+      transform: translateX(34px);
+    }
+  }
+}
+
+.switch-slider {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: $brand-red;
+  transition: 0.3s;
+  border-radius: 30px;
+
+  &::before {
+    position: absolute;
+    content: "";
```

Comment:
Do we need this here if it's not actually putting in any content?

Reply:
Yeah, I think pseudo-elements need this defined, otherwise they won't be inserted at all.